src/applications/medical-expense-report/config/chapters/01-applicant-information/veteranInformation.js

```diff
@@ -44,67 +46,18 @@ export function setDefaultIsOver65(oldData, newData, currentDate) {
 export default {
   updateFormData: setDefaultIsOver65,
   uiSchema: {
-    ...titleUI(
-      ({ formData }) =>
-        formData.claimantNotVeteran
-          ? 'Veteran information'
-          : 'Your information',
-    ),
-    veteranFullName: {
-      ...fullNameUI(),
-      first: {
-        'ui:title': 'First name',
-        'ui:required': formData => formData.claimantNotVeteran === true,
-        'ui:errorMessages': {
-          required: 'Enter a first name',
-        },
-        'ui:options': {
-          hideIf: formData => formData.claimantNotVeteran === false,
-        },
-      },
-      middle: {
-        'ui:title': 'Middle name',
-        'ui:options': {
-          hideIf: formData => formData.claimantNotVeteran === false,
-        },
-      },
-      last: {
-        'ui:title': 'Last name',
-        'ui:required': formData => formData.claimantNotVeteran === true,
-        'ui:errorMessages': {
-          required: 'Enter a last name',
-        },
-        'ui:options': {
-          hideIf: formData => formData.claimantNotVeteran === false,
-        },
-      },
-      suffix: {
-        'ui:title': 'Suffix',
-        'ui:options': {
-          hideIf: formData => formData.claimantNotVeteran === false,
-        },
-      },
-    },
-    veteranSocialSecurityNumber: {
-      ...ssnUI('Social Security number'),
-      'ui:errorMessages': {
-        pattern:
-          'Please enter a valid 9 digit Social Security number (dashes allowed)',
-        required: 'Enter a Social Security number',
-      },
-    },
+    ...titleUI('Veteran information'),
+    veteranFullName: conditionalVeteranNameUI(title => `Veteran's ${title}`),
```

Comment:
This file needs all of the changes, but the updated errorMessage text. 

- [ ] You can delete the helper file and use full name here too. 
- [ ] The title still needs to be conditional
- [ ] all the of the hide ifs need to still work
- [ ] remove veteran from any field names